src/sources/postgresql_metrics.rs

```diff
@@ -505,11 +540,7 @@ impl PostgresqlMetrics {
                     tags!(self.tags, "db" => db),
                 ),
             ]);
-            if row
-                .columns()
-                .iter()
-                .any(|column| column.name() == "checksum_failures")
-            {
+            if self.version.unwrap() >= 120000 {
```

Comment:
If we are going to leaving this `unwrap()` here for now, could we make it `expect("version is set above")`.